extra-admin/throttled/autobuild/build

```
@@ -0,0 +1,26 @@
+abinfo "Installing all scripts and configs ..."
+mkdir "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled -p
+install -Dvm755 "$SRCDIR"/throttled.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/throttled.py
+install -Dvm755 "$SRCDIR"/mmio.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/mmio.py
+install -Dvm644 "$SRCDIR"/etc/lenovo_fix.conf "$PKGDIR"/etc/throttled.conf
+install -Dvm644 "$SRCDIR"/systemd/lenovo_fix.service "$PKGDIR"/usr/lib/systemd/system/throttled.service
```

And for this case, you added verbosity, very good. But it's recommended that if the character numbers of a line is bigger than 80, insert a `\` into a proper place:

```suggestion
install -Dvm644 "$SRCDIR"/systemd/lenovo_fix.service \
    "$PKGDIR"/usr/lib/systemd/system/throttled.service
```

just like what you did at line 3, 4 and 5, 6.